components/offer/MakingOfferSingleItem.vue

```diff
@@ -88,10 +89,23 @@ const offerExpirationStoreItem = computed({
 })
 
 const highestOfferPrice = computed(() => formatWithBlank(Number(item.value.highestOffer) || 0) || '--')
+const trimmedHighestOfferPrice = computed(() => trimDuplicateZeroes(highestOfferPrice.value))
 
 const collectionFloorPrice = computed(() =>
   formatWithBlank(Number(item.value.collection.floorPrice?.[0]?.price || '0')) || '--',
 )
+const trimmedCollectionFloorPrice = computed(() => trimDuplicateZeroes(collectionFloorPrice.value))
+
+const trimDuplicateZeroes = (price: string): string => {
+  if (price === '--') {
+    return '--'
+  }
+  const [amount, chainSymbol] = price.split(' ')
+
+  const trimmedAmount = parseFloat(amount).toString()
```

Comment:
It's unnecessary to add another function and process the string. There is already `formatWithBlank` function. Could you handle it there? 

maybe
```
const formatWithBlank = (value: number) => {
  return value ?  `${parseFloat(formatBalance(value, decimals.value))} ${chainSymbol.value}`: '--'
}

```

Reply:
You are right, it's just a wrapper function that would be way cleaner. I'll change that